bundles/org.openhab.binding.elgatokeylight/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="elgatokeylight"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="keylight">
+		<label>Elgato Key Light</label>
+		<description>An Elgato Key Light</description>
+		<category>Lightbulb</category>
+
+		<channels>
+			<channel id="power" typeId="system.power" />
+			<channel id="brightness" typeId="system.brightness" />
+			<channel id="color_temperature" typeId="system.color-temperature" />
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host Address</label>
```

Comment:
You could add `<context>network-address</context>` to get a free config validation.